Makefile

```diff
@@ -1,6 +1,7 @@
 VERSION ?= $(shell ./get-versions.sh VERSION)
 REVISION ?= $(shell ./get-versions.sh REVISION)
 DIST ?= unstable
+DIST_VERSION ?= "~"$(shell sed '/CODENAME/!d;s/^.*=//' /etc/lsb-release)"1"
```

Comment:
Yes, lsb_release (the command) is required for calamari; if it's not explicit it should be